attestation-agent/attester/src/se/mod.rs

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) Copyright IBM Corp. 2024
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use super::Attester;
+use anyhow::*;
+
+pub mod ibmse;
+
+pub fn detect_platform() -> bool {
+    ibmse::is_se_guest()
+}
+
+#[derive(Debug, Default)]
+pub struct SeAttester {}
+
+#[async_trait::async_trait]
+impl Attester for SeAttester {
+    async fn get_evidence(&self, attestation_request: Vec<u8>) -> Result<String> {
+        // attestation_request is serialized SeAttestationRequest String bytes
+        ibmse::perform(&attestation_request)
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[ignore]
+    #[tokio::test]
+    async fn test_se_get_evidence() {
+        let attester = SeAttester::default();
+        let report_data: Vec<u8> = vec![0; 64];
+
+        let evidence = attester.get_evidence(report_data).await;
```

Comment:
I am afraid that this unit test cannot be run. As the input `report_data` will be treated as a `SeAttestationRequest` and the test here defines an empty one. Thus I suggest either
1. add a real worked `report_data` in under `https://github.com/confidential-containers/guest-components/tree/main/attestation-agent/attester/test_data` (requires a new dir creation)
2. delete this unit test